workers/Adarsh-NP/api_demo/app.py

```diff
@@ -0,0 +1,19 @@
+'''Simple python script to interact with the apis'''
```

Comment:
@abhinavbajpai2012 : Is this something you mean to deploy now, or is it from the earlier period and should be cleaned up? If so, the PR FROM `gsoc-dev` arrow pointed at your branch (opposite of this PR) FIRST, *should* clean that up.